rules/xcodeproj.bzl

```diff
@@ -265,6 +291,8 @@ $BAZEL_INSTALLER
             "$(clang_stub_short_path)": ctx.executable.clang_stub.short_path,
             "$(clang_stub_ld_path)": ctx.executable.ld_stub.short_path,
             "$(clang_stub_swiftc_path)": ctx.executable.swiftc_stub.short_path,
+            "$(infoplist_stub)": ctx.file._infoplist_stub.short_path,
+            "$(workspacesettings_xcsettings_short_path)": ctx.file._workspace_xcsettings.short_path,
```

Comment:
It is necessary to provide an Info.plist file which sets the bundle ID to $PRODUCT_BUNDLE_IDENTIFIER. Otherwise, if you try to run a test target which refers to a test host app and the test host app doesn't have an Info.plist, Xcode will complain that the Product bundle ID could not be determined.